node/clients/externalPrice.ts

```diff
@@ -25,6 +25,10 @@ export default class ExternalPrice
         return undefined
       }
 
+      if (e.response.status === 429) {
+        throw new TooManyRequestsError()
+      }
+
```

Comment:
The app doesn't have an error handler or something else, so when there is an exception, it throws the httpException, and the app returns 500 instead of 429. So it was needed to implicitly throw "TooManyRequestsError" and the app returns 429.